### DISCUSSION TITLE
[General] <What's the quickest way to get Test Eth from Console.optimism>

### DISCUSSION THREAD

**Discussion by KarlaGod** [ethereum-optimism/developers#542] [General]
### Discussion Topic

<img width="565" alt="image" src="https://github.com/user-attachments/assets/a4690922-f0c4-4bc7-ac30-2ab54731686f">
This is my Gitcoin Passport Score, yet it tells me I have no onchain identity and can't claim any test tokens.

<img width="758" alt="image" src="https://github.com/user-attachments/assets/5610f4c5-1287-443c-b8b6-6ebe28b00892">

What to do?

**Comment by imtipi** [comment]
i remember replied you before(did i?)
already reported to teams.
btw,I think both https://console.optimism.io/faucet https://app.optimism.io/faucet didn't verify gitcoin passport,will also report this💀

**Reply by KarlaGod** [reply]
No you didn't, this is the first time I'm asking about this here.

Thanks for reporting, could you let me know when the problem is fixed?

**Reply by imtipi** [reply]
nvm,it is a ticket which also related to the not working faucet yesterday
will notify you if team replies me back

**Comment by sim31** [comment]
Faucet is not working for me for a while now. Although, I'm not using gitcoin passport, just email login.

**Reply by hash1go** [reply]
Hello!
This issue has been reported to the OP team, so please wait for a while until it is resolved.

**Comment by a39955720** [comment]
Has this issue still not been resolved? I'm encountering the same problem now.

**Comment by Peace92** [comment]
Hello i have the same problem, even after clearing cache, it still does not recognize my gitcoin passport 
![77](https://github.com/user-attachments/assets/72669a52-4b9a-4940-9eb0-839af01dc986)
![onch](https://github.com/user-attachments/assets/91c6a684-dcca-4891-ae2c-a9c79d19ad17)

**Comment by ozgurgungor333** [comment]
I have the same issue, i have 37gtc passport score but it says: Your connected wallet doesn’t have onchain identity

**Reply by Peace92** [reply]
I hope that team will identify the cause and add a fix soon.

**Comment by azadeh84** [comment]
i have 38 point on gitcoin but i cant get op faucet and it shows that i dont have onchain idendity 
![Screenshot (801)](https://github.com/user-attachments/assets/914e407c-a5a4-467d-995c-82e3482ffd1c)
 what shall i do?

**Comment by azadeh84** [comment]
i have 38 point on gitcoin but i cant get op faucet and it shows that i dont have onchain idendity. i have this issue from july why nobody care about it. i hope team take an action for this problem soon.

**Reply by Peace92** [reply]
yeah same, still waiting for team to fix it

**Comment by rollinman** [comment]
Problems with identification. The data was not synchronized. My wallet is - 0x8A1113023472f8b4B3aF3Bc317eB19a5A115fC4C
![Screenshot 2025-03-07 162105](https://github.com/user-attachments/assets/8916de6a-2773-4150-b337-65a0c9f496f8)
![Screenshot 2025-03-07 162019](https://github.com/user-attachments/assets/e631cb9b-affa-4372-9634-98ecb3ef5632)
